evalml/data_checks/outliers_data_check.py

```diff
@@ -44,6 +44,19 @@ def validate(self, X, y=None):
             ...                   "details": {"columns": ["z"], "rows": [3], "column_indices": {"z": [3]}}}],
             ...     "errors": [],
             ...     "actions": [{"code": "DROP_ROWS", "metadata": {"rows": [3], "columns": None}}]}
+            ...
+            >>> assert OutliersDataCheck.get_boxplot_data(df['z']) == {
+            ...     'score': 0.888652177485398,
```

Comment:
I wonder if there's a way we can round so that we don't check so many significant figures. If gamma.cdf is off in the ten-thousands place in a future release we're cooked.